### PR TITLE
Reorder operations sections + some capitalization and rendering fixes

### PR DESCRIPTION
**Pull request by fgalan**
This PR is intended to reorder the sections inside each API operation, having the following sections inside each request:

- Request Query Parameters
- Request URL Parameters
- Request Payload
- Response

This PR is part of a bigger series https://github.com/telefonicaid/fiware-orion/pull/4141

### CODE REVIEW

File: doc/manuals/orion-api.md [hunk 1]
```diff
@@ -1189,30 +1191,39 @@ The values that `options` parameter can have for this specific request are:
...
-              }
-            }
-          }
-        }
+* Successful operation uses 200 OK
+* Errors use a non-2xx and (optionally) an error payload. See subsection on "Error Responses" for more details.
+
+Example Response 200:
+
+Content-Type is `application/json``
```
Comment (fgalan) [inline doc/manuals/orion-api.md:1201]:
```suggestion
Content-Type is `application/json`
```

File: doc/manuals/orion-api.md [hunk 2]
```diff
@@ -1343,39 +1360,41 @@ that `id` and `type` are not allowed.
...
-**Query Parameters**
+**Request Query Parameters**
 
 | Parameter  | Optional | Type   | Description                                                                                                                                                                                            | Example      |
 |------------|----------|--------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|
 | `type`     | ✓        | string | Entity type, to avoid ambiguity in case there are several entities with the same entity id.                                                                                                            | `Room`       |
 | `options`  | ✓        | string | Only `keyValues` option is allowed for this method. When used, the response payload uses the `keyValues` simplified entity representation. See "Simplified Entity Representation" section for details. | keyValues    |
 
-+ Request (application/json)
+**Request**
```
Comment (fgalan) [inline doc/manuals/orion-api.md:1378]:
```suggestion
**Request payload**
```

File: doc/manuals/orion-api.md [hunk 3]
```diff
@@ -1386,66 +1405,65 @@ that `id` and `type` are not allowed.
...
```
Comment (fgalan) [inline doc/manuals/orion-api.md:1423]:
```suggestion
**Request payload**
```

File: doc/manuals/orion-api.md [hunk 4]
```diff
@@ -1134,6 +1105,42 @@ The values that `options` parameter can have for this specific request are:
...
+
+* Successful operation uses 201 Created (if upsert option is not used) or 204 No Content (if
+  upsert option is used). Response includes a `Location` header with the URL of the
+  created entity.
+* Errors use a non-2xx and (optionally) an error payload. See subsection on "Error Responses" for
+  more details.
+
 + Response 201
 
     + Headers
```
Comment (mapedraza) [inline doc/manuals/orion-api.md:1146]: Should we leave this part like this? (examples of responses with just the header)

File: doc/manuals/orion-api.md [hunk 5]
```diff
@@ -1734,34 +1765,38 @@ This operation returns a JSON object with information about the type:
...
 * `count` : the number of entities belonging to that type.
 
-Response code:
+**Request query parameters**
+
+| Parameter    | Optional | Type   | Description  | Example |
+|--------------|----------|--------|--------------|---------|
+| `entityType` |          | string | Entity Type. | `Room`  |
+
+*Response*
```
Comment (mapedraza) [inline doc/manuals/orion-api.md:1774]:
```suggestion
**Response**
```

File: doc/manuals/orion-api.md [hunk 6]
```diff
@@ -1469,20 +1481,30 @@ Those parameter are part of the URL request. They are mandatory.
...
-          "type": "Number",
-          "metadata": {}
-        }
+* Successful operation uses 200 OK.
+* Errors use a non-2xx and (optionally) an error payload. See subsection on "Error Responses" for
+  more details.
+
+Example Response 200:
+
+Content-Type is `application/json``
```
Comment (fgalan) [inline doc/manuals/orion-api.md:1499]:
```suggestion
Content-Type is `application/json`
```

File: doc/manuals/orion-api.md [hunk 7]
```diff
@@ -1584,20 +1599,30 @@ Those parameter are part of the URL request. They are mandatory.
...
-          "city": "Madrid",
-          "country": "Spain"
-        }
+* Successful operation uses 200 OK.
+* Errors use a non-2xx and (optionally) an error payload. See subsection on "Error Responses" for
+  more details.
+
+Response 200 example:
+
+Content-Type is `application/json``
```
Comment (fgalan) [inline doc/manuals/orion-api.md:1616]:
```suggestion
Content-Type is `application/json`
```

File: doc/manuals/orion-api.md [hunk 8]
```diff
@@ -1974,100 +2014,103 @@ Response:
...
-            "attrs": ["temperature", "humidity"],
-            "timesSent": 12,
-            "lastNotification": "2015-10-05T16:00:00.00Z"
-            "lastSuccess": "2015-10-05T16:00:00.00Z"
-          },
-          "expires": "2025-04-05T14:00:00.00Z",
-          "status": "active",
-          "throttling": 5
-        }
+Content-Type is `application/json``
```
Comment (fgalan) [inline doc/manuals/orion-api.md:2033]:
```suggestion
Content-Type is `application/json`
```

File: doc/manuals/orion-api.md [hunk 9]
```diff
@@ -2156,81 +2199,88 @@ The values that `options` parameter can have for this specific request are:
...
 #### Create Registration [POST /v2/registrations]
 
 Creates a new context provider registration. This is typically used for binding context sources
 as providers of certain data.
 The registration is represented by a JSON object as described at the beginning of this section.
 
-Response:
+**Request payload** 
+
+Content-Type is `application/json``
```
Comment (fgalan) [inline doc/manuals/orion-api.md:2253]:
```suggestion
Content-Type is `application/json`
```

File: doc/manuals/orion-api.md [hunk 10]
```diff
@@ -2243,98 +2293,102 @@ Response:
...
-            "expires": "2017-10-31T12:00:00",
-            "status": "failed",
-            "forwardingInformation": {
-              "timesSent": 12,
-              "lastForwarding": "2017-10-06T16:00:00.00Z",
-              "lastFailure": "2017-10-06T16:00:00.00Z",
-              "lastSuccess": "2017-10-05T18:25:00.00Z",
-            }
-      }      
+Content-Type is `application/json``
```
Comment (fgalan) [inline doc/manuals/orion-api.md:2312]:
```suggestion
Content-Type is `application/json`
```

File: doc/manuals/orion-api.md [hunk 11]
```diff
@@ -2243,98 +2293,102 @@ Response:
...
 |------------------|--------|--------------------------------------|----------------------------|
 | `registrationId` | string | Id of the subscription to be updated | `62aa3d3ac734067e6f0d0871` |
 
-Response:
+**Request payload** 
 
-* Successful operation uses 204 No Content
-* Errors use a non-2xx and (optionally) an error payload. See subsection on "Error Responses" for
-  more details.
+Content-Type is `application/json``
```
Comment (fgalan) [inline doc/manuals/orion-api.md:2360]:
```suggestion
Content-Type is `application/json`
```

File: doc/manuals/orion-api.md [hunk 12]
```diff
@@ -2156,81 +2199,88 @@ The values that `options` parameter can have for this specific request are:
...
-            "http":{ 
-              "url": "http://localhost:1234"
-            }
-          }
-        }
-
-+ Response 201
+Example response 201:
 
     + Headers
```
Comment (mapedraza) [inline doc/manuals/orion-api.md:2286]: This part renders in a weird way

image

File: doc/manuals/orion-api.md [hunk 13]
```diff
@@ -1632,22 +1651,30 @@ Those parameter are part of the URL request. They are mandatory.
...
-+ Request (application/json)
+**Request payload**
 
-        {
-          "address": "Ronda de la Comunicacion s/n",
-          "zipCode": 28050,
-          "city": "Madrid",
-          "country": "Spain"
-        }
+Content-type is `application/json`.
```
Comment (fgalan) [inline doc/manuals/orion-api.md:1662]: This is the only request we have in the API that also allows text MIME type.

```suggestion
Content-type is `application/json` or `text/plain`
```

File: doc/manuals/orion-api.md [hunk 14]
```diff
@@ -1878,89 +1907,100 @@ The values that `options` parameter can have for this specific request are:
...
-            "attrs": ["temperature", "humidity"]
-          },            
-          "expires": "2025-04-05T14:00:00.00Z",
-          "throttling": 5
-        }
-
-+ Response 201
+Example Response 201:
 
     + Headers
```
Comment (mapedraza) [inline doc/manuals/orion-api.md:2006]: How this part is rendered is a bit weird

Same as https://github.com/telefonicaid/fiware-orion/pull/4156/files#r911902208

**Review by fgalan — COMMENTED** [review]
LGTM

**Pull request merged**